clang/test/SemaSYCL/intel-fpga-nofusion.cpp

```diff
@@ -0,0 +1,39 @@
+// RUN: %clang_cc1 -fsycl -fsycl-is-device -fsyntax-only -ast-dump -Wno-sycl-2017-compat -verify %s
+// expected-no-diagnostics
+
+void foo() {
+  int a1[10], a2[10];
+
+  // CHECK: AttributedStmt
+  // CHECK-NEXT: SYCLIntelFPGANofusionAttr {{.*}}
```

Comment:
It seems `FileCheck` command is missing, otherwise `CHECK` comments are not necessary here.

Reply:
Done